manual/sphinx/user_docs/coordinates.rst

```diff
@@ -1044,7 +1070,7 @@ From the definition
 Parallel derivative of the B field
 ----------------------------------
 
-To get the parallel nablaients of the `B` field components, start
```

Comment:
Incidentally, this was a googlewhack: the only instance of this word on google! Looks like it was the result of `s/grad/nabla/` :smile: